github/REVIEWING.md

```diff
@@ -0,0 +1,341 @@
+   You can try the command below and, if you get a message such as `remote ... already exists`, you can go to the next step.
+
+   ```console
+   git remote add <username> <URL>
+   ```
+
+   where `<username>` is the username in the URL, e.g. `cluosh`, `michpappas`.
+   E.g.:
+
+   ```console
+   git remote add cluosh http://github.com/cluosh/unikraft
+   git remote add michpappas https://github.com/michpappas/unikraft
+   ```
+
+   Verify it was added correctly by using:
+
+   ```console
+   git remote show <username>
+   ```
+
+   E.g.:
+
+   ```console
+   git remote show cluosh
+   git remote show michpappas
+   ```
+
+1. Check the remote branch that was used to create the pull request:
+
+   ```console
+   git branch -a
+   ```
+
+   In the output of the above command you should see a line `remotes/<username>/<branch>`.
+   If you don't see that, you did something wrong and you need to remove and recreate the remote, via the `git remote` command.
+
+1. Fetch the contribution from the remote end:
+
+   ```console
+   git fetch <username>
+   ```
+
+   E.g.:
+
+   ```console
+   git fetch cluosh
+   git fetch michpappas
+   ```
+
+1. Create a local branch of the contribution:
+
+   ```console
+   git checkout -b <local-branch-name> <username>/<branch>
+   ```
+
+   where:
+
+   - `<username>` is the name of the remote
+   - `<branch>` is the name of the remote branch
+   - `<local-branch-name>` is the name of the local branch you will use
+
+   The name of the local branch is something you choose.
+   Generally, it is advised to keep the same name as `<branch>`.
+   However, it may be the case that the contributor uses a value for `<branch>` that conflicts with an existing local one (such as `main` or `master` or `testing`).
+   In that case, it is suggested to use `<username>-<branch>` for the local branch name.
+
+   E.g.:
+
+   ```console
+   git checkout -b cluosh-staging cluosh/staging
+   git checkout -b michpappas/feature/libukmemtag michpappas/michpappas/feature/libukmemtag
+   ```
+
+   In case of the first command, we used `<username>-<branch>` for the local branch.
+   This is because the remote branch name (`staging`) conflicted with a local branch name called the same (`staging`).
+
+   In the case of the second command we used the same name for the local branch as for the remote branch: `michpappas/feature/libukmemtag`.
+   Note that there is a recommendation to use, as branch name, the format `<username>/<branch-detail>`, to be able to pin the branch name to the user creating it.
+
+So, at this point you should have:
+
+- Your browser pointing out to the pull request interface on GitHub.
+- Your local repository pointing to the contents of the remote branch used in the pull request.
+
+We will detail the 5 review action below, mentioning whether they will happen either on the GitHub pull request interface or locally, on the clone of the repository.
+
+## Review Commit Metadata
+
+Reviewing the commit metadata happens locally.
```

Comment:
Can be done on the web too. Either via https://github.com/razvand/snippets/pull/3/commits/50b9b2c5ffd8d65e6646edfb3dd1c70568fc2d63 (click on the commit in the PR), or https://github.com/razvand/snippets/commit/50b9b2c5ffd8d65e6646edfb3dd1c70568fc2d63.patch (add `.patch` at the end of the previous URL). Depends on what you want to check